lua/nvim-treesitter/configs.lua

```diff
@@ -48,6 +48,14 @@ local builtin_modules = {
     enable = false,
     is_supported = queries.has_indents,
   },
+  fold = {
+    module_path = "nvim-treesitter.fold",
+    do_not_attach = true,
+    fold_one_line_after = false,
```

Comment:
probably this should also allow a list/map of languages. So `true` or `{"python" =true, "cpp" = true}`

Reply:
i think maybe taking either a list or a boolean it would be better indeed.